src/app/PCM/templates/tpl-header/tpl-header-mobile/lojing/index.js

```diff
@@ -9,7 +9,8 @@ const TemplateHeaderMobile = ({ mappedPcm, depth }) => {
     <nav>
       <Flex align="center" paddingBlock="4" paddingInlineStart={14} paddingInlineEnd={4}>
         <Box mx={5} alignItems="center" m="auto">
-          <PcmField code={PRI_LOC1} mappedPcm={mappedPcm} depth={depth} />
+          <Image src={'/lojing-logo.png'} alt={''} w={'7.5rem'} />
+          {/* <PcmField code={PRI_LOC1} mappedPcm={mappedPcm} depth={depth} /> */}
```

Comment:
let's remove this piece of code if it is not being used rather than commenting it out, if we don't think we are going to use it in future.